backend/plonk/bls12-377/prove.go

```diff
@@ -696,13 +725,6 @@ func (s *instance) computeLinearizedPolynomial() error {
 
 func (s *instance) batchOpening() error {
 
-	// wait for LRO to be committed (or ctx.Done())
```

Comment:
Just to confirm - have you checked that this synchronization point was not necessary? Were you able to remove it because we have the next check for the linearizedPolynomial do be done?

Reply:
yes so in batchOpening we wait for `chLinearizedPolynomial` to be closed,  `computeLinearizedPolynomial` waits for `chH` to be closed, and `computeQuotient` (which closes chH) waits for `s.chLRO` to be closed